autofit/non_linear/nest/dynesty/plotter.py

```diff
@@ -1,13 +1,45 @@
 from dynesty import plotting as dyplot
+import logging
 
 from autofit.plot import SamplesPlotter
 from autofit.plot.samples_plotters import skip_plot_in_test_mode
 
+logger = logging.getLogger(__name__)
 
 class DynestyPlotter(SamplesPlotter):
+    
+    @staticmethod
+    def log_plot_exception(plot_name : str):
+        """
+        Plotting the results of a ``dynesty`` model-fit before they have converged on an
+        accurate estimate of the posterior can lead the ``dynesty`` plotting routines
+        to raise a ``ValueError``.
+
+        This exception is caught in each of the plotting methods below, and this
+        function is used to log the behaviour.
+
+        Parameters
+        ----------
+        plot_name
+            The name of the ``dynesty`` plot which raised a ``ValueError``
+        """
+
+        logger.info(

```

Comment:
Maybe make this a warning or throw an exception?

Reply:
Its expected behaviour (Dynesty cant visualize certain things mid-fit) so its better as info.